<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>label1</class>
 <widget class="QMainWindow" name="label1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Card Reader</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(238, 238, 238);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="front_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Front side</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>216</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Back side</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_imgf">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_imgb">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>584</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
